signaling/develop/call-invite-notification.mdx

```diff
@@ -8,21 +8,21 @@ description: >
 
 export const toc = [{}];
 
-To implement call notification, you need to integrate the Agora <Vg k="VSDK" />, the Agora <Vg k="SIG_SDK" />, and platform-specific call APIs such as ConnectionService for Android, CallKit for iOS, and CallKeep for Flutter and React Native. The <Vg k="SIG_SDK" /> supports call notification only when the app is running. So, you also need to integrate platform-specific APIs to ensure that users can still receive call notifications when the app is on the background or the process is closed.
+To implement call notification, you need to integrate the Agora <Vg k="VSDK" />, the Agora <Vg k="MESS_SDK" />, and platform-specific call APIs such as ConnectionService for Android, CallKit for iOS, and CallKeep for Flutter and React Native. The <Vg k="MESS_SDK" /> supports call notification only when the app is running. So, you also need to integrate platform-specific APIs to ensure that users can still receive call notifications when the app is on the background or the process is closed.
 
 ## Implementation
 
-### Step 1: Integrate the <Vg k="VSDK" /> and the <Vg k="SIG_SDK" />
+### Step 1: Integrate the <Vg k="VSDK" /> and the <Vg k="MESS_SDK" />
```

Comment:
As for the product name - single "l" or double "l"? Guess this is an easy issue to fix as all the product names are written in variables :)
<img width="1907" alt="image" src="https://user-images.githubusercontent.com/101544891/192244975-58e3e573-f408-46f6-b0d1-e486b8cfde9c.png">
<img width="1895" alt="image" src="https://user-images.githubusercontent.com/101544891/192245199-c6f2a47b-c744-4d45-9132-52625bb210a7.png">

Reply:
:-), done.